src/Model/lbsProcessor.py

```diff
@@ -82,14 +84,38 @@ def get_objects(self):
         """Return objects assigned to processor
         """
 
+        return self.objects.union([self.sentinel])
+
+    def get_migratable_objects(self):
+        """Return migratable objects assigned to processor
+        """
+
         return self.objects
 
+    def get_sentinel_object(self):
```

Comment:
In the new VT implementation, you will have a valid ID for every sentinel object. And you might have more than one, but you can collapse them if you want.

Reply:
Alright, so 2 follow-up questions:
* how is the valid ID generated. is there some logic that makes it predictable?
* shouldn't we then create a set of sentinel nodes (initially containing only one)?